os/arch/arm/src/armv7-a/arm_assert.c

```diff
@@ -583,24 +583,13 @@ void up_assert(const uint8_t *filename, int lineno)
 	}
 
 #ifdef CONFIG_SMP
-	/* If SMP is enabled and there is a crash in kernel space, then we need to
-	 * pause all the other cpu's immediately because the kernel state might be
-	 * invalid at this point. If we dont pause other cpu's then it might lead
-	 * to multiple asserts.
+	/* If SMP is enabled and there is a crash, then we need to pause all
+	 * the other cpu's immediately because the kernel state might be
+	 * invalid at this point. If we dont pause other cpu's then it might
+	 * lead to multiple asserts.
```

Comment:
I think this comment is not valid. Let us change as follows:
"Pause all other cpus to avoid mix up of logs while printing asssert logs"